tfhe/src/core_crypto/commons/math/random/gaussian.rs

```diff
@@ -98,6 +109,17 @@ where
             <Torus as FromTorus<f64>>::from_torus_custom_mod(s2, custom_modulus),
         )
     }
+
+    fn single_sample_success_probability(
+        distribution: Gaussian<f64>,
+        _modulus: Option<Self::CustomModulus>,
+    ) -> f64 {
+        // The modulus does not impact gaussian generation success
+        <(f64, f64) as RandomGenerable<Gaussian<f64>>>::single_sample_success_probability(
+            distribution,
+            None,
```

Comment:
More logical to pass modulus here IMO even if it is not used
Applies to other places

Reply:
you are right

Reply:
not sure what I was thinking there

Reply:
ok I know why it was done this way, the gaussian generation is done with f64 for a Torus, but the associated modulus is a Torus element, so I don't pass the modulus here otherwise it cannot compile, thankfully here it has no impact

Reply:
I'll update the comment to be clearer though as I could not recall why it was done this way